src/key.rs

```diff
@@ -31,7 +33,8 @@ impl DatabaseKeyIndex {
     }
 
     pub fn key_index(self) -> Id {
```

Comment:
Can we make this `const`?

Reply:
Done (though `const` doesn't actually have any effect unless the method is actually used in a `const`-context).